recipes/power-grid-model/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "power-grid-model" %}
-{% set version = "1.4.69" %}
+{% set version = "1.4.70" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/power-grid-model-{{ version }}.tar.gz
-  sha256: 40ba3f8e98b2e883f8f8015064c13fa2f148141c778df2511c4a952ecede07de
```

Comment:
this sounds like it's going to be a hassle to keep up to date. how can this be automated?

Reply:
There will be bot later which will update the version and hash automatically.